src/test/java/edu/kis/powp/jobs2d/events/SelectRunCurrentCommandOptionListener.java

```diff
@@ -21,7 +21,8 @@ public SelectRunCurrentCommandOptionListener(DriverManager driverManager) {
 	@Override
 	public void actionPerformed(ActionEvent e) {
 		DriverCommand command = CommandsFeature.getDriverCommandManager().getCurrentCommand();
-		CommandHistory.addEntry(CommandsFeature.getDriverCommandManager().getCommandList(),CommandsFeature.getDriverCommandManager().getCommandName());
+		if(!CommandHistory.logMode)
```

Comment:
Of course, this could be also implemented using observer. (Not necessary)

Reply:
Zostawiamy to tak jak jest